### ISSUE TITLE
Allow selection across transcript text block on YT annotator

### ISSUE THREAD

**Issue by dwhly** [dwhly-proj/droppdf#76]

**Comment by genuineBuildMonkey** [comment]
I think this works now (if I understand).

I'm able to select across blocks of text and annotate. The annotation highlights across blocks also.

**Comment by dwhly** [comment]
yes, but it includes the timestamp in the selection.  I've changed the name of this issue.

**Comment by nhan000** [comment]
Any updates since then. This can be really close to perfect.
